### PR TITLE
Testing1

### PR DESCRIPTION
**Pull request by janakthakkar1985**

### CODE REVIEW

File: test10 [hunk 1]
```diff
@@ -0,0 +1 @@
+Hlllo this is jnak from devops techlab.
```
Comment (devopstechlab) [inline test10]: change spelling

File: test2 [hunk 2]
```diff
@@ -1,2 +1,3 @@
 this is test file.
-what a file.
\ No newline at end of file
+what a file.
+updating file.
```
Comment (devopstechlab) [inline test2]: su lakhyu che baka

**Review by devopstechlab — CHANGES_REQUESTED** [review]
make required changes

File: test10 (hunk 1 above)
Comment (janakthakkar1985) [reply test10]: made required changes

File: test2 (hunk 2 above)
Comment (janakthakkar1985) [reply test2]: now perfect

**Review by devopstechlab — APPROVED** [review]
going to merge in master

**Pull request merged**